src/yb/yql/pggate/util/ybc_stat.cpp

```diff
@@ -88,6 +88,9 @@ ybcstat_get_wait_event_type(uint32_t wait_event_info) {
         case YB_FLUSH_AND_COMPACTION:
             event_type = "Flush and Compaction";
             break;
+        case YB_CQL_WAIT_STATE:
+            event_type = "CQL Server";
+            break;
```

Comment:
I don't see wait events for DMLRead and DMLWrite i.e. when CQLServer has sent an RPC to tserver.

Reply:
Adding these